src/ValidationMessage/index.jsx

```diff
@@ -0,0 +1,85 @@
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
+
+import newId from '../utils/newId';
+import styles from './ValidationMessage.scss';
+
+const inputProps = {
+  dangerIconDescription: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+  className: PropTypes.arrayOf(PropTypes.string),
+  errorId: PropTypes.string,
+  isValid: PropTypes.bool,
+  themes: PropTypes.arrayOf(PropTypes.string),
+  validationMessage: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+};
+
+const defaultProps = {
+  dangerIconDescription: '',
+  className: [],
+  errorId: undefined,
+  isValid: true,
+  themes: [],
+  validationMessage: '',
+};
+
+class ValidationMessage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const id = this.props.errorId ? this.props.errorId : `error-${newId('validationMessage')}`;
```

Comment:
There's an implicit dependency between the `error-{id}` format here and the format that's defined in the initial state declaration in `Fieldset` (aka, if we change something in one place we probably need to change it in the other place) - this seems kind've rigid?

Reply:
Nit: if you call this variable `errorId` then you can just do 

```javascript
this.state = { errorId };
```

on the next line

Reply:
I refactored this so that `ValidationMessage` has a required `id` prop and no state. `Fieldset` and `asInput` now create and pass the `error-{id}` themselves. I think that makes the most sense since `Fieldset` and `asInput` will have to use that same id in their `aria-describedby`.